crates/oxc_linter/src/rules/jsx_a11y/aria_role.rs

```diff
@@ -176,13 +157,13 @@ fn test() {
     use crate::tester::Tester;
 
     fn ignore_non_dom_schema() -> serde_json::Value {
-        serde_json::json!([2,{
```

Comment:
These are necessary to remove, otherwise the rule config doesn't work. I don't entirely know why this was working previously?